lib/auth/clt.go

```diff
@@ -1611,39 +1611,6 @@ func (c *Client) SetStaticTokens(st types.StaticTokens) error {
 	return nil
 }
 
-func (c *Client) GetAuthPreference() (types.AuthPreference, error) {
```

Comment:
This is just the endpoints being excised from the HTTP client, yeah? Are there any backwards compatibility issues to consider here?

EDIT: Have just seen the additions to `httpfallback` - does that cover it?

Reply:
Yes, the http fallback methods should cover compatibility concerns